content/rest/channels.textile

```diff
@@ -182,10 +184,14 @@ h3(#namespaces). Channel namespaces
 
 <%= partial 'shared/_channel_namespaces' %>
 
-h2. Presence
+h2(#presence). Presence
 
 Channels expose a <span lang="default">@presence@</span><span lang="csharp">@Presence@</span> member which a client can use to obtain present members and presence event history for the channel itself. See the "REST presence":/rest/presence documentation for details.
 
+h2(#channel-metadata). Channel Metadata
```

Comment:
As mentioned elsewhere, I think we need to better position this, explain it, and use consistent namgin.